packages/typespec-vscode/src/vscode-cmd/openapi3-preview.ts

```diff
@@ -154,7 +154,9 @@ async function loadOpenApi3PreviewPanel(
           }
           await clearOutputFolder(outputFolder);
 
-          const result = await client.compileOpenApi3(mainTspFile, srcFolder, outputFolder);
+          // If the path contains spaces, it will cause compilation failure, so quotes are required
+          const escapedMainTspFile = mainTspFile.includes(" ") ? `"${mainTspFile}"` : mainTspFile;
+          const result = await client.compileOpenApi3(escapedMainTspFile, srcFolder, outputFolder);
```

Comment:
please make the compileOpenApi3 to handle it so that the caller doesn't need to worry about this kind of problem